docs/docs/animations/withTiming.mdx

```diff
@@ -54,6 +54,26 @@ enum ReduceMotion {
 #### `toValue`
 
 The value on which the animation will come at rest.
+There are many possible types for `toValue`, currently the following kinds of values are supported:
+
+- **colors** - we support wide range of color formats:
+  - hex numbers - `0xff1234`
+  - rgb - `rgb(100,50,0)`
+  - rgba - `rgba(255,105,180,0)`
+  - hsl - `hsl(0, 50%, 50%)`
+  - hex strings - `#53575E`
+  - css color names - `dodgerblue`
+- **numbers**
+- **suffixed numbers** - strings being a number with a unit, like `"100%"` or `"90deg"`
```

Comment:
The "suffixed numbers" category isn't very precise. MDN specifies directly what "suffixed numbers" are supported and it's separated as https://developer.mozilla.org/en-US/docs/Web/CSS/angle or https://developer.mozilla.org/en-US/docs/Web/CSS/percentage

Are `150px` "suffixed number" supported? From this sentence I'm not sure.

Please check the source code and write precisely what types (angle, percentage, or more?) are supported.

Reply:
Sure, so technically we support all 'normal' literal suffixes applied with no space to the number.
"Normal" means here that neither "zł" or "$" works, the suffix should consist of letters only. I will change the message.

<details><summary> Example code </summary>

```jsx
import Animated, {
  useAnimatedProps,
  useSharedValue,
  withTiming,
  Easing,
} from 'react-native-reanimated';
import { Button, TextInput, StyleSheet, View } from 'react-native';

import React from 'react';

Animated.addWhitelistedNativeProps({ text: true });

const AnimatedTextInput = Animated.createAnimatedComponent(TextInput);

export default function CounterExample() {
  const suffix = 'PLN';
  const sv = useSharedValue(`0${suffix}`);
  const toggle = useSharedValue(false);

  const animatedProps = useAnimatedProps(() => {
    return { text: sv.value, defaultValue: sv.value };
  });

  const handleToggle = () => {
    const newValue = toggle.value ? `0${suffix}` : `10${suffix}`;
    toggle.value = !toggle.value;

    sv.value = withTiming(newValue, {
      duration: 1000,
      easing: Easing.steps(10, true),
    });
  };

  return (
    <>
      <View style={styles.buttons}>
        <Button onPress={handleToggle} title="Toggle" />
      </View>
      <AnimatedTextInput
        animatedProps={animatedProps}
        style={styles.text}
        editable={false}
      />
    </>
  );
}

const styles = StyleSheet.create({
  buttons: {
    marginVertical: 50,
  },
  text: {
    fontSize: 100,
    fontVariant: ['tabular-nums'],
    textAlign: 'center',
  },
});
```
</details>

https://github.com/software-mansion/react-native-reanimated/assets/56199675/12787d19-d693-4262-9d93-b663383cd0e8